content/docs/command-reference/pull.md

```diff
@@ -129,8 +129,7 @@ $ cd example-get-started
 
 </details>
 
-The workspace looks almost like in this
-[pipeline setup](/doc/tutorials/pipelines):
+The workspace looks almost like in this [pipeline setup](/doc/tutorials/dags):
```

Comment:
Same. Use /doc/start/data-pipelines or remove.

Reply:
I don't think the link is needed here. Same reason as `fetch.md`.

Reply:
I don't see a comment from you in the fetch review but OK, I agree let's just remove these. Please check any other instances to that link in docs. Thanks